Doc/library/sqlite3.rst

```diff
@@ -408,6 +258,154 @@ Module functions and constants
       UnraisableHookArgs(exc_type=<class 'ZeroDivisionError'>, exc_value=ZeroDivisionError('division by zero'), exc_traceback=<traceback object at 0x10b559900>, err_msg=None, object=<function <lambda> at 0x10b4e3ee0>)
       <sqlite3.Cursor object at 0x10b1fe840>
 
+.. function:: register_adapter(type, adapter, /)
+
+   Register an *adapter* callable to adapt the Python type *type* into an
+   SQLite type.
+   The adapter is called with a Python object of type *type* as its sole
+   argument, and must return a value of a
+   :ref:`type that SQLite natively understands <sqlite3-types>`.
+
+.. function:: register_converter(typename, converter, /)
+
+   Register the *converter* callable to convert SQLite objects of type
+   *typename* into a Python object of a specific type.
+   The converter is invoked for all SQLite values of type *typename*;
+   it is passed a :class:`bytes` object and should return an object of the
+   desired Python type.
+   Consult the parameter *detect_types* of
+   :func:`connect` for information regarding how type detection works.
+
+   Note: *typename* and the name of the type in your query are matched
+   case-insensitively.
+
+
+.. _sqlite3-module-constants:
+
+Module constants
+^^^^^^^^^^^^^^^^
+
```

Comment:
If you want, you could add a sentence that introduces the constants defined in the module, possibly mentioning the fact that their values shouldn't be changed even though it's technically possible to do it.

Reply:
> If you want, you could add a sentence that introduces the constants defined in the module [...]

My initial reaction is -0. I'll see if I can phrase something meaningful, though; if I can't, I won't add anything :)

> [...] possibly mentioning the fact that their values shouldn't be changed even though it's technically possible to do it.

I'm -1
No other module takes this approach, and IMO, adding such a sentence is kinda like pointing at the red button labelled _DO NOT PRESS_ 🔴 😄

Reply:
True, perhaps the sentence is unnecessary.  The reason why I suggested it is that the title is potentially misleading since it seems to imply that these are actually constants -- the sentence would help clarify that they aren't.  This is just nitpicking though, feel free to merge without extra sentences :)